rules/library.bzl

```diff
@@ -960,6 +961,9 @@ def apple_library(
         # To be backward compatible with rules_apple 2.x
         swift_kwargs = dicts.add(kwargs, {"plugins": plugins} if plugins else {})
 
+        # Allow consumers to opt-in to https://github.com/bazelbuild/rules_swift/pull/1162
+        swift_kwargs = dicts.add(swift_kwargs, {"always_include_developer_search_paths": swift_always_include_developer_search_paths} if swift_always_include_developer_search_paths else {})
```

Comment:
Want to do the if check first before mutating the `swift_kwargs` to minimize the penalty to all callers?

Reply:
Makes sense, we should do the same in the `plugins` line above in that case

Reply:
https://github.com/bazel-ios/rules_ios/pull/852/commits/71de457039f733c1b871f15e3d8f5842939647c1